### DISCUSSION TITLE
Which language models to add support for

### DISCUSSION THREAD

**Discussion by victor-hugo-dc** [sinaptik-ai/pandas-ai#58] [General]
I imagine you want to add support for [Google's Bard](https://bard.google.com/) and [Stanford's Alpaca](https://crfm.stanford.edu/2023/03/13/alpaca.html), any other LLMs that you had in mind?

**Comment by gventuri** [comment]
Hey @victor-hugo-dc!

I'd say also [Claud by Anthropic](https://www.anthropic.com/index/introducing-claude).
I'm also integrating [Starcoder](https://github.com/bigcode-project/starcoder), almost done!

**Comment by gventuri** [comment]
@mzeidhassan suggested to have a look at this: https://chat.lmsys.org/

**Comment by duotangx** [comment]
@gventuri any plans to include support for claude models outside amazon bedrock?

**Comment by ulloogeo** [comment]
I would also vote for adding support for Cohere's models
